CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 0.9.2
+
+* Fixed an issue where implementations supporting Python 3.10 but not the peer certificate chain APIs
```

Comment:
mention GraalPy specifically?

Reply:
That makes sense, I'll add that.

Reply:
Done in https://github.com/sethmlarson/truststore/pull/150/commits/c322ba42eb9f9a2892e769d100f721c627a57a33